src/mvnw.js

```diff
@@ -38,30 +39,41 @@ function shell() {
 /**
  * Run mvnw with provided commands.
  * @param {Hash} args - All arguments to pass to it
+ * @return {Promise} of maven execution task
  */
 module.exports = function(args) {
-  const home = path.resolve(__dirname, '../mvnw');
-  const bin = path.resolve(home, 'mvnw') + (process.platform == 'win32' ? '.cmd' : '');
-  const params = args.filter(function(t) {
-    return t != '';
-  }).concat([
-    '--errors',
-    '--batch-mode',
-    '--update-snapshots',
-    '--fail-fast',
-  ]);
-  const cmd = bin + ' ' + params.join(' ');
-  console.debug('+ %s', cmd);
-  const result = spawnSync(
-    bin,
-    process.platform == 'win32' ? params.map((p) => `"${p}"`) : params,
-    {
-      cwd: home,
-      stdio: 'inherit',
-      shell: shell(),
-    }
-  );
-  if (result.status != 0) {
-    throw new Error('The command "' + cmd + '" exited with #' + result.status + ' code');
-  }
+  return new Promise((resolve, reject) => {
+      const home = path.resolve(__dirname, '../mvnw');
+      const bin = path.resolve(home, 'mvnw') + (process.platform == 'win32' ? '.cmd' : '');
+      const params = args.filter(function(t) {
+        return t != '';
+      }).concat([
+        '--errors',
+        '--batch-mode',
+        '--update-snapshots',
+        '--fail-fast',
+      ]);
+      const cmd = bin + ' ' + params.join(' ');
+      console.debug('+ %s', cmd);
+      const result = spawn(
```

Comment:
@volodya-lombrozo somewhere you indent by two spaces, somewhere by four. I'm surprised `eslint` doesn't catch you.

Reply:
@yegor256 fixed. I also have added new [issue](https://github.com/objectionary/eoc/issues/99) for that.